package.json

```diff
@@ -91,7 +91,8 @@
     "pkg": "node ./scripts/pkg.js",
     "polykey": "ts-node src/polykey.ts",
     "start": "ts-node src/polykey.ts -- agent start --verbose",
-    "dev": "nodemon src/polykey.ts -- agent start --verbose"
+    "dev": "nodemon src/polykey.ts -- agent start --verbose",
+    "postshrinkwrap": "./scripts/npmDepsHash.sh"
```

Comment:
I thought there was a different script for this? I don't remember `postshrinkwrap` anywhere? Are you sure this is correct.

Reply:
Wasn't it supposed to be the `dependencies` script?

Also you want to group the npm native scripts together.

Reply:
Fixed in 1d7af25.